Sources/TuistLoader/Models+ManifestMappers/RunActionOptions+ManifestMapper.swift

```diff
@@ -13,11 +13,15 @@ extension TuistGraph.RunActionOptions {
         generatorPaths: GeneratorPaths
```

Comment:
Do you mind creating a test file similar [to this one](https://github.com/tuist/tuist/blob/main/Tests/TuistLoaderTests/Models%2BManifestMappers/CodeCoverageMode%2BManifestMapperTests.swift) where we test that the properties are properly mapped?